kerchunk/kerchunk-in-practice.ipynb

```diff
@@ -0,0 +1,2704 @@
+{
```

Comment:
<p>Should reading come first? How will someone know, or do they need to know it's Kerchunk vs Zarr?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
I was thinking to have creating come first, for consistency